hw_oop/ex2.py

```diff
@@ -0,0 +1,25 @@
+class Animal:
+    animal_type = 'Any animal'
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+
+    def say(self):
+        return "Hi, I am {} {} and I am {} !".format(self.animal_type, self.name, self.age)
+        # return f"Hi, I am {self.name} and I am {self.age}!"
+
+
+class Dolphin(Animal):
+    animal_type = "Dolphin"
+
+
+class Zebra(Animal):
+    animal_type = "Zebra"
+
+
+d = Dolphin(age=20, name="Chuck")
+z = Zebra("Zelda", 30)
+
+print(d.say())
+print(z.say())
```

Comment:
This red circle means there is no new line at the end of file. Use PyCharm build in formatter to aavoid this mistakes.